docs/release-notes.md

```diff
@@ -35,7 +35,7 @@ New Features:
 -  airframe-http: Support Scala.js for defining Web interfaces as well as Scala JVM [[0e017db](https://github.com/wvlet/airframe/commit/0e017db)]
 -  airframe-http: Support adding a custom codec to clients ([#1004](https://github.com/wvlet/airframe/issues/1004)) [[9675669](https://github.com/wvlet/airframe/commit/9675669)]
 -  airframe-http: Add cross-platform HttpMessage.Request and Response classes ([#982](https://github.com/wvlet/airframe/issues/982)) [[f55d438](https://github.com/wvlet/airframe/commit/f55d438)]
--  airframe-http-finagle: Update finagle-core, finagle-http, ... to 20.3.0 ([#965](https://github.com/wvlet/airframe/issues/965)) [[111e5ad](https://github.com/wvlet/airframe/commit/111e5ad)]
+-  airframe-http-finagle: Update finagle-core, finagle-http, ... to 20.4.0 ([#965](https://github.com/wvlet/airframe/issues/965)) [[111e5ad](https://github.com/wvlet/airframe/commit/111e5ad)]
```

Comment:
We must not update the past release notes. This line must be reverted.

Reply:
Nice catch :)

Reply:
I didn't know scala-steward changes such release notes too. I'll be more careful next time 😓

Reply:
Yeah, I was really surprised that scala-steward can change this part.